### ISSUE TITLE
Left handed mode

### ISSUE THREAD

**Issue by Snyret** [Divested-Mobile/Mull-Fenix#155]
Please add the option to switch the bar elements to the left (home page to the right and tabs, three dots to the left) together with the close buttons in the compact mode.

**Comment by SkewedZeppelin** [comment]
yep yep, out of scope: https://github.com/mozilla-mobile/fenix/issues/6365
